### DISCUSSION TITLE
Change parameters of an existing element

### DISCUSSION THREAD

**Discussion by CoolJoe86** [e2nIEE/pandapower#1453] [Q&A]
Hello, I am wondering if it is possible to change the parameters of an element after it has been created.

Specifically, I am trying to optimize cable capacity in a grid, so I would like to alter the line std_type between runs. Is this possible, and if so what is the syntax to achieve this?

Thanks!

**Comment by rbolgaryn** [comment]
Hi @CoolJoe86,

You can change the parameters either individually, as in a normal pandas DataFrame, or use the function change_std_type.

There are some helpful functions in pandapower.toolbox that you can take a look at.

Roman

**Reply by CoolJoe86** [reply]
Excellent, thank you!
